yarn.lock

```diff
@@ -0,0 +1,2070 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
don't commit this please (I would be okay with a commit adding this to `.gitignore`)